### DISCUSSION TITLE
steam deck support or guide?

### DISCUSSION THREAD

**Discussion by xsessive182** [abraunegg/onedrive#1925] [Q&A] [answered]
Hi guys ,

I recently got a steam deck and wanted to check if its possible to make this app run there. From what i have seen steam deck prefers flatpaks.

**Comment by abraunegg** [comment, marked as answer]
@xsessive182 
Steam deck is based on arch linux - so if you can access the steam deck base os, you can try and install from source or from the arch aur

There is no flatpak as they are for gui applications
